src/ontology/config/icd10cm_exclusion_reasons.robot.template.tsv

```diff
@@ -0,0 +1,173 @@
+term_id	exclusion_reason
+ID	AI rdfs:seeAlso
```

Comment:
instead of seeAlso, we should use https://github.com/monarch-initiative/mondo/issues/5177 when its done

Reply:
Sounds good! I added a TODO above the appropriate line for now and referenced that issue.

Reply:
Sounds good! I added a TODO for now.

Reply:
Twice I responded to this and my comments (including like 5-10 others) were deleted! I just saw a red banner at the top of the page I must not have noticed earlier, ugh.

Anyway sure thing. I made a TODO for now.

Reply:
ok third time ive added a comment and its not showing up. wth is happening